README.md

```diff
@@ -125,7 +125,6 @@ WARNING: You have not specified enough hosts to satisfy the blueprint.
 ```
 $ yaas cluster create cluster1 --template=template1.json
 $ cat template2.json | yaas cluster create cluster2
-$ yaas cluster create cluster3
```

Comment:
Look at the doc you posted--you still have to post JSON, which is essentially a cluster creation template. It just doesn't have a blueprint/hosts but instead has a version.